### DISCUSSION TITLE
Groundwork Attribution in Footer

### DISCUSSION THREAD

**Discussion by krowvin** [USACE/groundwork#63] [Ideas]
I think we should consider some sort of attribution in the footer. 

Reason being, this helps differentiate Groundwork from the AFPIMS site and the vanilla templates that exist. 

It also provides a means to others to go and try out Groundwork for themselves. 

_if_ we did this, questions i'd have are:

1. Should we force the attribution? Or make a flag to disable?
2. Should it link to the groundwork homepage, or github? 
3. Should it be barely visibile, or more in the way like the existing DMA attributions? (See Footer [here](https://usace.army.mil/))

**Comment by willbreitkreutz** [comment]
That's not a bad idea, I hadn't really thought about it.  I'd probably put it in the footer with a flag you could turn on if you wanted, similar to the logos

**Comment by adamscarberry** [comment]
Could we put the version of groundwork in the footer to see at a glance if folks are use an old outdated release?  Probably somewhere out of the way and unnoticeable.  

Another topic of "how do we notify folks when new releases are available" made me think of this.  For that solution, I think they can just use Custom Watch settings:
![image](https://github.com/USACE/groundwork/assets/46494851/41eaf70f-f7ec-44aa-861d-7e7612fa648b)
